packages/stats-py/Dockerfile

```
@@ -0,0 +1,35 @@
+WORKDIR /opt/aqsha-stats
+COPY pyproject.toml uv.lock README.md ./
+
+# Pin EXACT resolved versions from uv.lock so every snapshot build is byte-for-byte
+# the same stack — the "deterministic SPSS-parity" guarantee. `pip install .` alone
+# floats deps to whatever the index serves that day (scipy/statsmodels/pingouin p-values
+# shift in the 3rd-4th decimal → verdicts flip at the p=0.05 boundary; golden tests stay
+# green against a different local env). Export the lock to pinned requirements and
+# install BEFORE copying source, so code edits don't bust the heavy dependency layer.
+# `--group sandbox` menyertakan openpls-engine (GPL) yang hanya hidup di image ini,
+# bukan sebagai dependency package.
+RUN pip install uv \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)